app/scripts/controllers/rif/rns/register/index.js

```diff
@@ -180,4 +182,32 @@ export default class RnsRegister extends RnsJsDelegate {
     return `${dataSignature}${dataOwner}${dataSecret}${dataDuration}${dataName}`;
   }
 
+  /**
+   * Overrides the parent operation because rns-js is failing on this, we invoke the contract directly.
+   */
+  createSubdomain (domainName, subdomain, ownerAddress, parentOwnerAddress) {
+    domainName = this.addRskSuffix(domainName);
+    if (!ownerAddress) {
+      ownerAddress = this.address;
```

Comment:
what is the idea here?

why, if the owner of the subdomain is not set, then the owner is the associated with the `RnsRegister`?

I think that doesnt specifying a ownerAddress must be an error

Reply:
The requirement said that the ownerAddress is optional, if the ownerAddress is not set then the ownerAddress is the ownerAddress from the parent domain, that's why is not an error.

Reply:
So why ownerAddress isnt parentOwnerAddress then?

Reply:
That's true, good catch